data-science-onramp/data-cleaning/clean_test.py

```diff
@@ -0,0 +1,158 @@
+import os
+import re
+import uuid
+
+from google.cloud import dataproc_v1 as dataproc
+from google.cloud import storage
+import pytest
+
+# Set global variables
+PROJECT = os.environ['GCLOUD_PROJECT']
+REGION = "us-east4"
```

Comment:
What is this region for? Is it for dataproc? A different resource?  Plz make the variable name reflect what resource it represents

Reply:
I believe this is for a dataproc cluster and I have updated it as such. Please correct me if I'm wrong @Symmetries